### ISSUE TITLE
UnicodeDecodeError in execute()

### ISSUE THREAD

**Issue by vladimir-avinkin** [m4ce/megacli-python#15]
```
  File "/opt/Python-3.6.2/lib/python3.6/site-packages/megacli/__init__.py", line 41, in execute
    out = out.decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 79: invalid start byte
```

I'm not sure how non utf8 character found it's way into megacli output, but it did. 
I think it would be sufficient to add `replace="ignore"` to `decode()`. I can submit PR if you wish

**Comment by m4ce** [comment]
@mouzfun, please feel free to open a PR.

**Comment by m4ce** [comment]
Released in 0.0.11.
